content/bn/blog/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: কুবারনেটস ব্লগ
+linkTitle: ব্লগ
+menu:
+  main:
+    title: "ব্লগ"
+    weight: 40
+    post: >
+       <p>কুবারনেটেস এবং সাধারণভাবে কন্টেইনার স্পেসের জন্য সাম্প্রতিক সংবাদ পড়ুন, এবং প্রেস থেকে সরাসরি প্রাপ্ত প্রযুক্তিগত হাউ-টুস পেতে থাকুন।</p>
+---
+{{< comment >}}
+
+ব্লগে কন্ট্রিবিউটিং সম্পর্কিত তথ্যের জন্য, দেখুন
+https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/#write-a-blog-post
+
+{{< /comment >}}
```

Comment:
```suggestion
```

You can miss this out if you like.

Reply:
@sftim do you have to add this part after we localize [this](https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/#write-a-blog-post) ?

Reply:
@sftim Do we need to add this part after localizing [this](https://kubernetes.io/docs/contribute/new-content/blogs-case-studies/#write-a-blog-post)?